src/features/feedback/components/FeedbackComment.tsx

```diff
@@ -16,7 +16,11 @@ interface FeedbackCommentProps {
 export const FeedbackComment = ({ post }: FeedbackCommentProps) => {
   const prettyDate = useMemo(() => {
     if (!post) return ''
-    return format(new Date(post.createdAt), 'dd MMM yyyy, hh:mmaaa')
+    return formatInTimeZone(
```

Comment:
on second look, since this is on the client, no need actually format in time zone. The Date created will be in local time and `format` should not affect the timezone.